vita3k/gui/src/controllers_dialog.cpp

```diff
@@ -366,6 +369,8 @@ void draw_controllers_dialog(GuiState &gui, EmuEnvState &emuenv) {
 
                     ImGui::End();
                 }
+                ImGui::TableSetColumnIndex(2);
+                ctrl.controllers_has_motion_support[i] ? ImGui::Text("%s", "Yes") : ImGui::Text("%s", "Yes");
```

Comment:
```suggestion
                ctrl.controllers_has_motion_support[i] ? ImGui::Text("%s", common["yes"].c_str()) : ImGui::Text("%s", common["no"].c_str());
```

Reply:
No need using 2 time ImGui::Text, set condition if inside it after , "%s", condition ? Yes : No